specification/source/glossary.rst

```diff
@@ -0,0 +1,50 @@
+.. _sec_glossary:
+
+Glossary
+--------
+
+This section provides definitions for abbreviations and terms introduced in
+the Open Building Controls project.
+
+.. If you add new entries, keep the alphabetical sorting.
+
+.. glossary::
+
+   Controls Description Language
+     The Control Description Language (CDL) is the language
+     that is used to express control sequences and requirements.
```

Comment:
A requirement is something that is tested and either passes, fails, or is untested. For example, a requirement would be that the actual actuation signal is within 2% of the signal computed using the CDL representation of a sequence, provided that they both receive the same input data. See also www.ep.liu.se/ecp/118/067/ecp15118625.pdf